tests/js/spec/components/organizations/timeRangeSelector/__snapshots__/dateSummary.spec.jsx.snap

```diff
@@ -22,12 +22,12 @@ exports[`DateSummary renders 1`] = `
             <div
               className="css-1gi0kmy-Date eyxmd2r2"
             >
-              Oct 13, 2017
+              Oct 14, 2017
               <Time>
                 <div
                   className="css-ul1hyh-Time eyxmd2r3"
                 >
-                  22:38
+                  04:38
```

Comment:
Is this on purpose? Did you run `yarn test` (vs `jest` directly)? `yarn test` is a wrapper to `jest` that sets some env vars including timezone.

Reply:
I did not change this intentional, not even sure why this updated.